backend/src/main/scala/cromwell/backend/standard/StandardLifecycleActorFactory.scala

```diff
@@ -181,12 +182,20 @@ trait StandardLifecycleActorFactory extends BackendLifecycleActorFactory {
 
   override def getExecutionRootPath(workflowDescriptor: BackendWorkflowDescriptor, backendConfig: Config,
                                     initializationData: Option[BackendInitializationData]): Path = {
-    initializationData.get.asInstanceOf[StandardInitializationData].workflowPaths.executionRoot
+    initializationData match {
+      // Try to use the initializationData if present to avoid recreating a WorkflowPath
+      case Some(data) => data.asInstanceOf[StandardInitializationData].workflowPaths.executionRoot
+      case None => new WorkflowPathsWithDocker(workflowDescriptor, backendConfig).executionRoot
```

Comment:
Actually in this case we could just call `super.getWorkflowExecutionRootPath` I think.
It doesn't really matter as of now anyway since it won't be called anymore if initialization fails so we should always have a Some(data)